README.md

```diff
@@ -107,6 +107,10 @@ LOGGING_LEVEL is INFO by default. Allowed logging levels are NOTSET, DEBUG, INFO
 
 ```bash
 conda create -n rag-test python=3.11
```

Comment:
I would change this env `rag-test` name to `rag-experiment` (the rest of the appearances as well)
```suggestion
conda create -n rag-experiment python=3.11
```